### PR TITLE
Remove useless icons on Android

### PR DESCRIPTION
**Pull request by dannywillems**
Related to https://github.com/ocsigen/ocsigen-start/issues/257.
iOS not done because I can test for the moment.

### CODE REVIEW

File: template.distillery/Makefile.mobile [hunk 1]
```diff
@@ -379,13 +379,29 @@ $(IOS_ICONS_PATH)/icon.png: $(IOS_ICONS_PATH)/icon-57.png; cp $< $@
...
-# Android icons
-$(ANDROID_ICONS_PATH)/ldpi.png: $(ANDROID_ICONS_PATH)/icon-36.png; cp $< $@
-$(ANDROID_ICONS_PATH)/mdpi.png: $(ANDROID_ICONS_PATH)/icon-48.png; cp $< $@
-$(ANDROID_ICONS_PATH)/hdpi.png: $(ANDROID_ICONS_PATH)/icon-72.png; cp $< $@
-$(ANDROID_ICONS_PATH)/xhdpi.png: $(ANDROID_ICONS_PATH)/icon-96.png; cp $< $@
-$(ANDROID_ICONS_PATH)/xxhdpi.png: $(ANDROID_ICONS_PATH)/icon-144.png; cp $< $@
-$(ANDROID_ICONS_PATH)/xxxhdpi.png: $(ANDROID_ICONS_PATH)/icon-192.png; cp $< $@
+$(ANDROID_ICONS_PATH)/ldpi.png: $(ANDROID_ICONS_PATH)/icon-36.png
+	cp $< $@
+	$(RM) $<
```
Comment (vasilisp) [inline template.distillery/Makefile.mobile]: That seems weird. Next time we need `ldpi.png`, its dependency `$(ANDROID_ICONS_PATH)/icon-36.png` will be missing, so `make` will re-generate something we previously deleted (and we will repeat the same work).

Comment (dannywillems) [inline template.distillery/Makefile.mobile]: Yes. You're right. Instead, I think I will change the icon names in `config.xml`.

**Comment by dannywillems** [comment]
So I changed. I removed ldpi, mdpi, hdpi, ... and use directly icon-36, icon-48, icon-72, etc in config.xml.

**Comment by vasilisp** [comment]
Thanks! Much nicer this way.

**Pull request merged**